src/api/parser/env.local

```diff
@@ -32,3 +32,12 @@ FEED_QUEUE_DELAY_MS=60000
 FEED_QUEUE_PARALLEL_WORKERS=1
 
 PARSER_PORT = 10000
+
+################################################################################
+# Supabase Services
+################################################################################
+
+SUPABASE_URL=http://localhost/v1/supabase
+
+# Supabase Secrets
+ANON_KEY=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyAgCiAgICAicm9sZSI6ICJhbm9uIiwKICAgICJpc3MiOiAic3VwYWJhc2UtZGVtbyIsCiAgICAiaWF0IjogMTY0MTc2OTIwMCwKICAgICJleHAiOiAxNzk5NTM1NjAwCn0.dc_X5iR_VP_qT0zsiyj_I_OZ2T9FtRU2BBNWN8Bu4GE
```

Comment:
@humphd, we are thinking about using the staging/production database to get all feeds in the parser. What do you think?

If so, we need to change this public anon key to one we use on deployment.

Reply:
If so, I'll wait for Dave to change both the `SUPABASE_URL` and the `ANON_KEY`

Reply:
Add a comment that you're using the staging anon key here (in the comment above)